### PR TITLE
Backend tests

### PR DESCRIPTION
**Pull request by LBarenschee**

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -35,6 +35,12 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-resolver-dns-native-macos</artifactId>
+            <version>4.1.77.Final</version>
+            <classifier>osx-x86_64</classifier>
+        </dependency>
```
Comment (Flooooooooooorian) [inline backend/pom.xml:43]: You can remove this again.

File: backend/src/test/java/de/neuefische/backend/controller/APIControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package de.neuefische.backend.controller;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class APIControllerTest {
+
```
Comment (Flooooooooooorian) [inline backend/src/test/java/de/neuefische/backend/controller/APIControllerTest.java:6]: I think there is something missing here

File: backend/src/test/java/de/neuefische/backend/service/APIServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class APIServiceTest {
+
+
+    @Test
+    public void checkIf_apiKey_isValid(){
```
Comment (Flooooooooooorian) [inline backend/src/test/java/de/neuefische/backend/service/APIServiceTest.java:11]: Same here

**Review by Flooooooooooorian — APPROVED** [review]

File: backend/src/test/java/de/neuefische/backend/controller/APIControllerTest.java (hunk 2 above)
Comment (LBarenschee) [reply backend/src/test/java/de/neuefische/backend/controller/APIControllerTest.java:6]: only prepared the test class. still have to write the tests

File: backend/src/test/java/de/neuefische/backend/service/APIServiceTest.java (hunk 3 above)
Comment (LBarenschee) [reply backend/src/test/java/de/neuefische/backend/service/APIServiceTest.java:11]: same as above

**Pull request merged**